### PR TITLE
Add a stdio option for debugging

### PR DESCRIPTION
**Pull request by paulkernfeld**
This could help us to debug the OSX build failures on Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,6 @@ language: node_js
 node_js:
   - "0.10"
   - "4"
+  - "5"
```
Comment (kesla) [inline .travis.yml]: why? 5 isn't supported anymore

File: test/test.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ setTimeout(function() {
 }, 2000);
 
 var options = {
-	display: {width: 1024, height: 980}
+	display: {width: 1024, height: 980},
+  stdio: 'inherit'
```
Comment (kesla) [inline test/test.js]: indentation looks off

**Comment by kesla** [comment]
Two comments :)

**Comment by paulkernfeld** [comment]
Sorry for the delay, I wrote the changes offline and forgot to push them!

File: .travis.yml (hunk 1 above)
Comment (paulkernfeld) [inline .travis.yml]: I did not know that! Thanks!

File: test/test.js (hunk 2 above)
Comment (paulkernfeld) [inline test/test.js]: Oops. I fixed the indentation of line 9 to match the rest of the file.

File: .travis.yml [hunk 3]
```diff
@@ -6,3 +6,4 @@ node_js:
   - "0.10"
   - "4"
   - "6"
+  - "node"
```
Comment (kesla) [inline .travis.yml]: this is smart :)

File: readme.md [hunk 4]
```diff
@@ -34,14 +34,18 @@ headless(200, function(err, childProcess, servernum) {
 });
 ````
 
-the icing on the cake : headless support optionals display parameters (width & height & depth) as well as arbitrary Xvfb arguments.
+Options:
+- `display`: optional display parameters (`width`, `height` & `depth`). If you want to specify `display`, `width` and `height` are required, but `depth` is optional and defaults to 16.
+- `args`: an array of arbitrary additional Xvfb arguments
+- `stdio`: set the stdio of the spawned Xvfb process(es). Set this to `'inherit'` to see Xvfb's stdout and stderr for debugging purposes.
```
Comment (kesla) [inline readme.md]: This is great, thanks a lot

**Review by kesla — APPROVED** [review]

**Pull request merged**